templates/actors/npc-sheet-2.hbs

```diff
@@ -0,0 +1,511 @@
+                        <span class="label roboto-upper">{{ localize "DND5E.LairActionLabel" }}</span>
+                        <span class="value">{{ system.resources.lair.initiative }}</span>
+                    </label>
+                    {{/if}}
+                    {{/if}}
+                </div>
+                {{/if}}
+
+            </div>
+
+        </div>
+
+    </header>
+
+    {{!-- Body --}}
+    <section class="sheet-body">
+
+        {{!-- Main Content --}}
+        <div class="main-content">
+
+            {{!-- Sidebar Collapser --}}
+            <button type="button" class="sidebar-collapser unbutton interface-only"
+                {{#if sidebarCollapsed}}
+                data-tooltip="JOURNAL.ViewExpand" aria-label="{{ localize "JOURNAL.ViewExpand" }}"
+                {{else}}
+                data-tooltip="JOURNAL.ViewCollapse" aria-label="{{ localize "JOURNAL.ViewCollapse" }}"
+                {{/if}}>
+                <i class="fas fa-caret-{{#if sidebarCollapsed}}right{{else}}left{{/if}}"></i>
+            </button>
+
+            {{!-- Sidebar --}}
+            <div class="sidebar">
+
+                {{!-- Speed --}}
+                <div class="pills-group {{#unless speed}}empty{{/unless}}">
+                    <h3 class="icon">
+                        <i class="fas fa-shoe-prints"></i>
+                        <span class="roboto-upper">{{ localize "DND5E.Speed" }}</span>
+                        {{#if editable}}
+                        <a class="config-button" data-action="movement" data-tooltip="DND5E.MovementConfig"
+                           aria-label="{{ localize "DND5E.MovementConfig" }}">
+                            <i class="fas fa-cog"></i>
+                        </a>
+                        {{/if}}
+                    </h3>
+                    <ul class="pills">
+                        {{#each speed}}
+                        <li class="pill">
+                            {{#each icons}}
+                            <i class="{{ icon }}" data-tooltip="{{ label }}" aria-label="{{ label }}"></i>
+                            {{/each}}
+                            <span class="label">{{ label }}</span>
+                            {{#if value}}
+                            <span class="separator">&vert;</span>
+                            <span class="value">{{ value }}</span>
+                            {{/if}}
+                        </li>
+                        {{/each}}
+                    </ul>
+                </div>
+
+                {{!-- Skills --}}
+                <div class="pills-group {{#unless skills}}empty{{/unless}}">
+                    <h3 class="icon">
+                        <i class="fas fa-briefcase"></i>
+                        <span class="roboto-upper">{{ localize "DND5E.Skills" }}</span>
+                        {{#if editable}}
+                        <a class="config-button" data-action="skills" data-tooltip="DND5E.SkillsConfig"
```

Comment:
I'm assuming this is not implemented just cause we don't have a "Configure All Skills" application?

Reply:
Yeah I left it out of this PR to reduce the line count and allow it to be reviewed separately.